src/components/Picklist/index.js

```diff
@@ -142,6 +144,11 @@ class Picklist extends Component {
         return undefined;
     }
 
+    getAriaActivedescendant() {
+        const { selectedOptionName, activeOptionName } = this.state;
+        return activeOptionName || selectedOptionName;
```

Comment:
here you can check whether there is not activeOptionName and then get the AriaActivedescendant form the value prop.
Also you need to figure out what take precedence in case when there is an option selected an an item has visual focus in this case you are giving priority to the option with visual focus